treetime/clock_tree.py

```diff
@@ -390,15 +388,28 @@ def _cleanup():
                     # Cx.y is the branch length corresponding the optimal subtree
                     bl = node.branch_length_interpolator.x
                     x = bl + node.date_constraint.peak_pos
-                    node.joint_pos_Lx = Distribution(x, node.branch_length_interpolator(bl),
-                                                     min_width=self.min_width, is_log=True)
+                    if hasattr(self, 'merger_model') and self.merger_model:
+                        node.joint_pos_Lx =  Distribution(x, -self.merger_model.integral_merger_rate(node.date_constraint.peak_pos) 
```

Comment:
this just adds a constant, right?

Reply:
Yes, this is the exact definition of the Lx distribution at the terminals. I noticed that adding it lead to less inconsistencies between this version and the current master branch later on, but is in theory not needed.